compact/far_forward/ZDC_SiPMonTile.xml

```diff
@@ -0,0 +1,120 @@
+    <constant name="ZDC_SiPMonTile_rotateY_angle" value="0*deg"/>
+    <constant name="ZDC_SiPMonTile_rotateZ_angle" value="ionCrossingAngle"/>
+
+    <constant name="ZDC_SiPMonTile_r_pos" value="35.8*m"/>
+    <constant name="ZDC_SiPMonTile_x_pos" value="sin(ZDC_SiPMonTile_rotateZ_angle)*ZDC_SiPMonTile_r_pos"/>
+    <constant name="ZDC_SiPMonTile_y_pos" value="0*m" />
+    <constant name="ZDC_SiPMonTile_z_pos" value="cos(ZDC_SiPMonTile_rotateZ_angle)*ZDC_SiPMonTile_r_pos"/>
+  </define>
+
+  <limits>
+  </limits>
+
+  <regions>
+  </regions>
+
+  <display>
+  </display>
+
+  <detectors>
+      <documentation>
+      ### SiPM-on-tile Zero-Degree Calorimeter Hcal
+
+      Each of the layers includes air gaps (front and back of each layer),
+      ESR foil (front and back of scintillator), a PCB, and an aluminum scitnillator cover
+
+    </documentation>
+    <detector
+      id="ZDC_SiPMonTile_ID"
+      name="ZDC SiPM-on-Tile Hcal"
+      type="ZeroDegreeCalorimeterSiPMonTile"
+      readout="ZDC_SiPMonTile_Hits"
+    >
+      <position x="ZDC_SiPMonTile_x_pos"         y="ZDC_SiPMonTile_y_pos"         z="ZDC_SiPMonTile_z_pos"/>
+      <rotation x="ZDC_SiPMonTile_rotateX_angle" y="ZDC_SiPMonTile_rotateY_angle" z="ZDC_SiPMonTile_rotateZ_angle"/>
+      <dimensions
+        x="ZDC_SiPMonTile_width"
+        y="ZDC_SiPMonTile_height"
+        z="ZDC_SiPMonTile_SingleLayerThickness*ZDC_SiPMonTile_Layer_NSteelRepeat +
+           ZDC_SiPMonTile_BackplateThickness"
+      />
+      <comment> Slices will be ordered according to the slice order listed here </comment>
+      <comment> Steel/Sc layers </comment>
+      <layer repeat="ZDC_SiPMonTile_Layer_NSteelRepeat" thickness = "ZDC_SiPMonTile_SingleLayerThickness">
+        <slice name="Absorber_slice" material="Steel235" thickness="ZDC_SiPMonTile_AbsorberThickness" vis="AnlGray"/>
+        <slice name="Air_slice" material="Air" thickness="ZDC_SiPMonTile_AirThickness"/>
+        <slice name="ScintCover_slice" material="Aluminum" thickness="ZDC_SiPMonTile_ScintillatorCoverThickness"/>
+        <slice name="ESRFoil_slice" material="Polystyrene" thickness="ZDC_SiPMonTile_ESRFoilThickness"/>
+        <slice name="Scintillator_slice" material="Polystyrene" thickness="ZDC_SiPMonTile_PolystyreneThickness" vis="AnlOrange" sensitive="true"/>
+        <slice name="ESRFoil_slice" material="Polystyrene" thickness="ZDC_SiPMonTile_ESRFoilThickness"/>
+        <slice name="PCB_slice" material="Fr4" thickness="ZDC_SiPMonTile_PCBThickness"/>
+        <slice name="Air_slice" material="Air" thickness="ZDC_SiPMonTile_AirThickness"/>
+      </layer>
+      <comment> Final layer of steel </comment>
+      <layer repeat="1" thickness = "ZDC_SiPMonTile_BackplateThickness">
+        <slice name="Absorber_slice" material="Steel235" thickness="ZDC_SiPMonTile_BackplateThickness" vis="AnlGray"/>
+      </layer>
+    </detector>
+  </detectors>
+
+  <readouts>
+    <readout name="ZDC_SiPMonTile_Hits">
```

Comment:
May I bring up naming for the output collections?
I was eyeing at Far-Forward calorimetry for a while (https://github.com/eic/EICrecon/issues/597), where there are some reversed suffixes compared to the central calorimetry. To me, the <class=Hcal><location=FarForward><subsystem=ZDC> scheme would be most friendly to people looking to start with ePIC by looking at what the data has to offer.
```suggestion
    <readout name="HcalFarForwardZDC_SiPMonTile_Hits">
```

Reply:
Thanks for the feedback @veprbl .   I have changed this, as you have requested, in commit [c175ebf].(https://github.com/eic/epic/commit/c175ebf86178bf6625b82e4e2f50639651192eee)